advanced/editor-command-identifiers.md

```diff
@@ -53,6 +53,7 @@ The commands on the following table are provided by the {{site.productname}} edi
 | Subscript            | Toggles subscript formatting to selection.                                                                                                                                                                                                                                                  |
 | Cut                  | Cuts the selected contents and puts in into users clipboard.                                                                                                                                                                                                                                |
 | Copy                 | Copies the selected contents and puts in into users clipboard.                                                                                                                                                                                                                              |
+| ContentLang          | Sets the language of the current selection. The value passed should be an object like those in [Content appearance options - `content_langs`]({{site.baseurl}}/configure/editor-appearance/#content_langs).                                                                                 |
```

Comment:
This is incorrect. The command name is `Lang` not `ContentLang`.
```suggestion
| Lang          | Sets the language of the current selection. The value passed should be an object like those in [Content appearance options - `content_langs`]({{site.baseurl}}/configure/editor-appearance/#content_langs).                                                                                 |
```

Reply:
I believe we use "passed in" as well.